.travis.yml

```diff
@@ -29,9 +31,8 @@ before_install:
   - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
       conda install --yes configparser ;
     fi
-  # Coding standards and tests.
-  - conda install --yes pytest
-  - conda install --channel conda-forge --yes pycodestyle flake8-print flake8-quotes flake8-import-order flake8-builtins flake8-comprehensions flake8-mutable flake8-docstrings
+  # Coding standards.
+  - conda install --channel conda-forge --yes flake8-print flake8-quotes flake8-import-order flake8-builtins flake8-comprehensions flake8-mutable
```

Comment:
Removed `flake8-docstrings` to avoid unnecessary noise and place the mandatory test requirements in the `requirement-dev.txt`